### PR TITLE
Inequality bugfix 2

### PR DESCRIPTION
**Pull request by mhramani**
Fix energy calculation when the lower bound (lb) equals the upper bound (lb) in ``add_linear_equality_constraint``.
This will resolve issue #914

### CODE REVIEW

**Comment by arcondello** [comment]
Can we add a test?

**Comment by hbarovertwo** [comment]
> Can we add a test?

Agree, a regression test will be great.

**Comment by rahmani15** [comment]
yes, I'll add a test later today.

File: releasenotes/notes/fix_inequlity_energy-0adf721256a75a30.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fix energy calculation when the lower bound (lb) equals the upper bound (lb)
+    in ``add_linear_equality_constraint``.
```
Comment (arcondello) [inline releasenotes/notes/fix_inequlity_energy-0adf721256a75a30.yaml:5]: Can we update this to be ``` ``DQM.add_linear_equality_constraint``` ``

Comment (arcondello) [reply releasenotes/notes/fix_inequlity_energy-0adf721256a75a30.yaml:5]: Or better ``` ``DiscreteQuadraticModel.add_linear_equality_cosntraint`` ```

Comment (rahmani15) [reply releasenotes/notes/fix_inequlity_energy-0adf721256a75a30.yaml:5]: it is for both BQM and DQM. I'll add both `DiscreteQuadraticModel.add_linear_equality_cosntraint`` and `BinaryQuadraticModel.add_linear_equality_cosntraint``

Comment (arcondello) [reply releasenotes/notes/fix_inequlity_energy-0adf721256a75a30.yaml:5]: You're totally, right. Perfect, thanks.

**Review by arcondello — APPROVED** [review]

**Review by hbarovertwo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**